diff --git a/node_modules/@astrojs/db/dist/runtime/index.js b/node_modules/@astrojs/db/dist/runtime/index.js
index 3034dad..2fc271e 100644
--- a/node_modules/@astrojs/db/dist/runtime/index.js
+++ b/node_modules/@astrojs/db/dist/runtime/index.js
@@ -15,10 +15,11 @@ const dateType = customType({
     return "text";
   },
   toDriver(value) {
-    if (value instanceof Date) return value.toISOString();
-    if (typeof value === 'number') return new Date(value).toISOString();
-    if (typeof value === 'string') return value;
+  if (!value) return null;
+  if (typeof value === 'string' || typeof value === 'number') {
     return new Date(value).toISOString();
+  }
+  return value.toISOString();
   },
   fromDriver(value) {
     if (!isISODateString(value)) {
